packages/experimental-nextjs-app-support/README.md

```diff
@@ -154,6 +158,60 @@ export default function RootLayout({
 
 If you want to make the most of the streaming SSR features offered by React & the Next.js App Router, consider using the [`useSuspenseQuery`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usesuspensequery_experimental) and [`useFragment`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usefragment_experimental) hooks.
 
+### Preloading data in RSC for usage in Client Components
+
+You can also preload data in RSC to populate the cache of your Client Components.
+
+For that, follow the setup steps for both RSC and Client Components as laid out in the last two paragraphs.
```

Comment:
```suggestion
For that, follow the setup steps for both RSC and Client Components as laid out in the last two sections.
```

Technically the first sentence in this section is a paragraph and I don't think you're meaning to reference that one, so I'd use the word "sections" here instead to avoid confusion.